### DISCUSSION TITLE
Student names not showing on Assignment page

### DISCUSSION THREAD

**Discussion by UTD-MrPeterson** [community/community#50724] [GitHub Education] [answered]
There was a website update that I received in the last 24 hours (first seen 3/22; the update includes more searching and sorting options) that also stopped showing student names for individual assignments. Instead I'm seeing their GitHub username (without the '@') in bold and the username repeated below that. I verified that my students' names and usernames are still linked, and that I can search for a name that doesn't match the username.

While I can get around this, it makes grading much more difficult.

(I'd attach a screenshot, but that would mean showing my students' login names, which I can't do.)

**Comment by sondregronas** [comment]
I'm experiencing the same. Also occurs in the new Classroom Assistant.

Would much rather prefer it if student names took precedence over usernames wherever possible, especially when cloning repos.

**Comment by christofmarti** [comment]
Same here. It is very annoying because we depend on the student name when evaluating results.
**It does not make sense to show the GitHub username twice.**

**Comment by liberty-bradleyd** [comment]
Same here. It might come as a surprise that I don't memorize all 160 of my student's github usernames, but I DO memorize their names. This change makes it way more difficult for understanding student progress.

**Comment by UTD-MrPeterson** [comment]
A workaround is that you can search for students' names. But still annoying.

I'm hoping it's just that the wrong field was hooked up to the bolded text.

**Comment by UTD-MrPeterson** [comment, marked as answer]
It's working for me now.

**Reply by liberty-bradleyd** [reply]
Ditto here.

Thanks whoever fixed it!
